wagtail_live/receivers.py

```diff
@@ -48,13 +52,30 @@ def is_embed(text):
 class BaseMessageReceiver:
     """Base Receiver class."""
 
-    def __init__(self, app_name, model_name):
+    @cached_property
+    def model(self):
         """
         LivePageMixin is an abstract class, so we can't make queries directly
-        We have to get the actual page which subclasses it to perform queries.
+        We have to get the actual model which subclasses it to perform queries.
         """
 
-        self.model = apps.get_model(app_name, model_name)
+        error_msg = "You won't be able to use Wagtail Live features without this setting defined."
+
+        app_name = getattr(settings, "LIVE_APP", "")
+        if not app_name:
+            logger.warning(
```

Comment:
**Suggestion:** since we're dealing with Django settings, wouldn't it make sense to raise `ImproperlyConfigured` with the message instead of logging a warning? Docs: https://docs.djangoproject.com/en/3.2/ref/exceptions/#improperlyconfigured

Reply:
Yes I agree with you that it will make more sense. We can do that.